Menu/Menu.js

```diff
@@ -8,8 +8,35 @@ let menuItems = [
   'Music',
   'Log Out'
 ];
+const header = document.querySelector('.header')
+const menuButton = document.querySelector('.menu-button')
 
-/* 
+function createMenuComponent(menuItems) {
+  const menu = document.createElement('div')
+  menu.classList.add('menu')
+
+  const list = document.createElement('ul')
+  list.classList.add('list')
+
+  menuItems.forEach(element => {
+    const listItem = document.createElement('li')
+    list.appendChild(listItem)
+    listItem.textContent = element
+  });
+
+  menu.appendChild(list)
+
+
+  menuButton.addEventListener('click', event => {
+    console.log('am I working')
```

Comment:
I love seeing debugging conosole logs. When I know I am done with my project, I like to go through and remove them so my code is clean before it gets merged into 'production'